### ISSUE TITLE
https://norussian.xyz/ is down

### ISSUE THREAD

**Issue by Leo32345** [jeremy-rutman/stop-russian-desinformation.near.page_source#2]
https://norussian.xyz/ is down for me.

A suggestion: Use Cloudflare DDoS protection to help prevent the site from being attacked. Cloudflare has free and paid plans, DDoS protection is included in the free plan.

**Comment by jeremy-rutman** [comment]
yeah they were actually using cloudflare for ru origins but looks like they were dos'ed in any case.  anyway you can dl the html and open it  locally , you don't need norussian.xyz or any other site

**Comment by Leo32345** [comment]
oh
that stinks

Honestly this is kind of ironic
